Exercise4.js

```diff
@@ -0,0 +1,5 @@
+function findEvenNumbers(numbers) {
+    const EVEN_NUMBERS = numbers.filter(element => element % 2 === 0)
+    return EVEN_NUMBERS
```

Comment:
1. Over usage of capital letters
2. is `EVEN_NUMBERS` needed? what about just returning its value immediately?